content/03-type-system.md

```diff
@@ -475,41 +475,41 @@ In this example, we are actually importing a **module**, not just a specific typ
-* If an enum is imported (directly or as part of a module import), all its [enum constructors](types-enum-constructor) are also imported (this is what allows the `OpAdd` usage in the above example).
+* If an enum is imported (directly or as part of a module import), all of its [enum constructors](types-enum-constructor) are also imported (this is what allows the `OpAdd` usage in the above example).
 
 Furthermore, it is also possible to import [static fields](class-field) of a class and use them unqualified:
 
 [code asset](assets/Import3.hx)
 
-Special care has to be taken with field names or local variable names that conflict with a package name: Since they take priority over packages, a local variable named `haxe` blocks off usage of the entire `haxe` package.
+Special care has to be taken with field names or local variable names that conflict with a package name. Since fields and local variables take priority over packages, a local variable named `haxe` blocks off usage of the entire `haxe` package.
 
 ##### Wildcard import
 
-Haxe allows using `.*` to allow import of all modules in a package, all types in a module or all static fields in a type. It is important to understand that this kind of import only crosses a single level as we can see in the following example:
+Haxe allows using a wildcard symbol `.*` to allow import of all modules in a package, all types in a module, or all static fields in a type. It is important to understand that this kind of import only crosses a single level as we can see in the following example:
 
 [code asset](assets/ImportWildcard.hx)
 
-Using the wildcard import on `haxe.macro` allows accessing `Expr` which is a module in this package, but it does not allow accessing `ExprDef` which is a sub-type of the `Expr` module. This rule extends to static fields when a module is imported.
+Using the wildcard import on `haxe.macro` allows accessing `Expr`, which is a module in this package, but it does not allow accessing `ExprDef` which is a sub-type of the `Expr` module. This rule extends to static fields when a module is imported.
 
-When using wildcard imports on a package the compiler does not eagerly process all modules in that package. This means that these modules are never actually seen by the compiler unless used explicitly and are then not part of the generated output.
+When using wildcard imports on a package, the compiler does not eagerly process all modules in that package; modules that have not been used explicitly are not part of the generated output. 
 
 ##### Import with alias
 
-If a type or static field is used a lot in an importing module it might help to alias it to a shorter name. This can also be used to disambiguate conflicting names by giving them a unique identifier.
+If a type or static field is used frequently in an imported module it might help to alias it to a shorter name. This can also be used to disambiguate conflicting names by giving them a unique identifier.
```

Comment:
I think the original is hard to understand, but the revision has a different meaning. The "importing module" is the module that has `import` lines. If we use a particular type often in the actual code (say, `import my.pack.FooBarWithALongName;`), the original is suggesting aliasing it might be helpful (`import my.pack.FooBarWithALongName as Foo;`).

Reply:
Sure, my bad for the information change.